assets/styles/components/atoms/SfButton.scss

```diff
@@ -7,7 +7,7 @@
     var(--font-weight--normal),
     var(--font-size--base),
     1.2,
-    var(--font-family--secondary)
+    var(--font-family--primary)
```

Comment:
do we need to give font family as primary? it won't be a default one?

Reply:
assigned primary font to variable `--button-font: var(--font-family--primary);` and used in `@include font()`.